### PR TITLE
Show different networks on validator explorer

### PR DESCRIPTION
**Pull request by Pedro-vk**
## Description

_Show different networks on validator explorer and be able to switch between them_

## Commit Message
Show different networks on validator explorer

### CODE REVIEW

File: packages/web/env-config.js.template [hunk 1]
```diff
@@ -11,6 +11,9 @@ const environments = {
     BLOCKSCOUT: {
       pageLength: 5,
       uri: 'http://localhost:4000/graphiql',
+      mainnet: 'http://localhost:4000/graphiql',
+      baklava: 'http://localhost:4000/graphiql',
```
Comment (aaronmgdr) [inline packages/web/env-config.js.template:15]: opinion: I know they were here before, but im not sure these URLs need to be in the ENV config file. only real purpose would be to either obscure them or to have different environments point to different networks but i think that is a recipe for not noticing a network isnt working until production.

File: packages/web/src/dev/ValidatorsList.tsx [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+import Button, { BTN } from 'src/shared/Button.3'
+import menu from 'src/shared/menu-items'
+import OvalCoin from 'src/shared/OvalCoin'
 import { colors, standardStyles, textStyles } from 'src/styles'
 import { weiToDecimal } from 'src/utils/utils'
 
+const networkMenu = [
+  // ['Mainnet', menu.VALIDATORS_LIST.link],
+  ['Baklava', menu.VALIDATORS_LIST__BAKLAVA.link],
+  ['Baklavastaging', menu.VALIDATORS_LIST_BAKLAVASTAGING.link],
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsList.tsx]: I believe we will want alfajores here too based on what @timmoreton said

File: packages/web/src/dev/ValidatorsList.tsx [hunk 3]
```diff
@@ -266,6 +289,26 @@ class ValidatorsList extends React.PureComponent<ValidatorsListProps & I18nProps
...
+            <View style={[styles.links]}>
+              {networkMenu.map(([name, link]) => (
+                <View
+                  key={name}
+                  style={[
+                    styles.linkWrapper,
+                    this.props.router.pathname !== link ? styles.linkWrapperInactive : {},
+                  ]}
+                >
+                  <Button kind={BTN.DARKNAV} href={link} text={name} />
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsList.tsx]: btw
consider extending https://github.com/celo-org/celo-monorepo/blob/master/packages/web/src/shared/navigation.tsx to take an optional coinColor Prop. and using.

**Comment by aaronmgdr** [comment]
Playing around on the preview site Im wondering if we can have the 
```
"Validator Explorer"
[list of networks]
```

part be rendered outside of the Apollo so when switching back and forth only the data gets refreshed. 

could be a different PR. 

what do you think?

File: packages/web/env-config.js.template (hunk 1 above)
Comment (Pedro-vk) [reply packages/web/env-config.js.template:15]: How do you think that we should handle that?

File: packages/web/src/dev/ValidatorsList.tsx (hunk 2 above)
Comment (Pedro-vk) [reply packages/web/src/dev/ValidatorsList.tsx]: Yes, I'm waiting for the rest of the information like mainnet to finish this part. Right now is more for feedback purposes.

File: packages/web/env-config.js.template (hunk 1 above)
Comment (aaronmgdr) [reply packages/web/env-config.js.template:15]: the network => url object could be kept either in the ValidatorList component or its own file.

unless you see a benefit of having them inside the ENV file? 

up to you.

Comment (Pedro-vk) [reply packages/web/env-config.js.template:15]: It's true that probably we won't need different variables by environment. So ok, I'll move to the component

**Comment by Pedro-vk** [comment]
> Playing around on the preview site Im wondering if we can have the
> 
> ```
> "Validator Explorer"
> [list of networks]
> ```
> 
> part be rendered outside of the Apollo so when switching back and forth only the data gets refreshed.
> 
> could be a different PR.
> 
> what do you think?

It's a good idea :muscle:

File: packages/web/src/dev/ValidatorsList.tsx (hunk 3 above)
Comment (Pedro-vk) [reply packages/web/src/dev/ValidatorsList.tsx]: I didn't see this component, so will be good to use it.

I think that I'm going to work on a new PR with this change and move the header outside the GraphQL query.

Comment (aaronmgdr) [reply packages/web/src/dev/ValidatorsList.tsx]: sweet

**Comment by Pedro-vk** [comment]
@aaronmgdr I moved the menu to the parent component to improve the UX and used Navigation component :+1:

File: packages/web/src/dev/ValidatorsList.tsx [hunk 4]
```diff
@@ -291,20 +295,14 @@ class ValidatorsList extends React.PureComponent<Props, State> {
...
-                  ]}
-                >
-                  <Button kind={BTN.DARKNAV} href={link} text={name} />
-                  {this.props.router.pathname === link && (
-                    <View style={styles.activeTab}>
-                      <OvalCoin color={colors.gold} size={10} />
-                    </View>
-                  )}
+              {networkMenuList.map(([name, link, navigate]: any) => (
+                <View key={name} style={[styles.linkWrapper]}>
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsList.tsx]: nit: unneeded array around linkWrapper

File: packages/web/src/shared/navigation.tsx [hunk 5]
```diff
@@ -4,26 +4,35 @@ import Button, { BTN } from 'src/shared/Button.3'
...
+  LIGHT,
+  DARK,
+}
+
 interface Props {
   link?: string
   onPress?: () => void
   text: string
   selected: boolean
+  theme?: NavigationTheme
```
Comment (aaronmgdr) [inline packages/web/src/shared/navigation.tsx:17]: btw I think from a design perspective there is no reason that you couldnt have gold and light or green on dark. 

but we can always tackle that if time comes.

**Review by aaronmgdr — APPROVED** [review]

File: packages/web/src/shared/navigation.tsx (hunk 5 above)
Comment (Pedro-vk) [reply packages/web/src/shared/navigation.tsx:17]: I did in this way because is necessary to set the button theme too

**Pull request merged**